obelisk/cpp/zoo/hardware/robots/unitree/CMakeLists.txt

```diff
@@ -6,27 +6,44 @@ message(STATUS "Configuring Unitree Interface")
 find_package(ament_cmake REQUIRED)
 
 include(FetchContent)
+
+# Let `unitree_sdk` refer to the content fetched from the git repo.
 FetchContent_Declare(
     unitree_sdk
     GIT_REPOSITORY https://github.com/unitreerobotics/unitree_sdk2.git
-    GIT_TAG 3a4680ae9b00df59e60f7e63cfb0fcc432a9d08d) # main # TODO: Fix the issue here
- 
-set(UNITREE_SDK_BUILD_EXAMPLES OFF CACHE BOOL "Disable building examples")
+    GIT_TAG 7b00ca4b766272d137cf7ff751d4a067bc99ca85) # Most recent commit includes the Write()
```

Comment:
Given we aren't including the D1 hardware interface, do you still need the other functions? 

I think we should be able to effectively leave this file untouched given we aren't merging in the D1 changes.